frontend/tests/playwright/test-fixtures.ts

```diff
@@ -35,9 +36,14 @@ export const test = base.extend<
 >({
    customServer: [
       // eslint-disable-next-line no-empty-pattern
-      async ({}, use) => {
+      async ({}, use, workerInfo) => {
+         process.env.NEXT_DIST_DIR = process.env.CI
+            ? './.next'
+            : `./.next-worker-${workerInfo.workerIndex}`;
          const props: CustomNextjsServer = await Server();
          await use(props);
+         // Clean up the dist directory after the worker is done.
+         exec('rm -rf .next-worker-*');
```

Comment:
I would think we'd want / need an await here as this is an async process.

Reply:
Although `exec` is an `async` process, we are not doing anything with the return value of `exec` nor are we running dependent logic after it. Therefore, the `await` would have no effect.

![Await Not Needed](https://user-images.githubusercontent.com/22064420/218543675-3f277632-ff88-4712-8d8c-cbb78e3c864d.png)

Reply:
I'd double-check whether the test runner could potentially quit before this finishes running, or whether it'll block properly for this. Could change the command to a `sleep` possibly to just force it to wait.

Reply:
Andrew captured what I was referring to but it seems like we're safe here as node will wait on outstanding invocations (of `exec` at least):
![image](https://user-images.githubusercontent.com/26855/218589375-edf9e2cf-14b0-4d48-9140-193507e7f5e9.png)